nilearn/datasets/atlas.py

```diff
@@ -889,10 +976,25 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
     see :footcite:`AAL_atlas`,
     and :footcite:`TZOURIOMAZOYER2002273`.
 
+    .. warning::
+
+        The maps image (``data.maps``) contains 117 unique integer values
+        defining the parcellation. However, these values are not consecutive
+        integers from 0 to 116 as is usually the case in Nilearn.
+        Therefore, these values shouldn't be interpreted as indices for the
+        list of label names.
+        For example, to get the name of the region corresponding to label
```

Comment:
Here also can we reserve "label" for strings/ region names and use "Region IDs" (or anything else you can think of) for integers in the map?